code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -0,0 +1,76 @@
+/datum/species/ethereal
+	name = "Ethereal"
+	id = "ethereal"
+	attack_verb = "burn"
+	attack_sound = 'sound/weapons/slash.ogg'
+	miss_sound = 'sound/weapons/slashmiss.ogg'
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/lizard
+	exotic_bloodtype = "Liquid Electricity" //fuck you think of something better gamer
+	siemens_coeff = 0.5 //They thrive on energy
+	brutemod = 1.25 //They're weak to punches
+	attack_type = BURN //burn bish
+	hair_color = "mutcolor"
+	damage_overlay_type = null //We are too cool for regular damage overlays
```

Comment:
don't null this, use "". Like Plasmemes:

https://github.com/tgstation/tgstation/blob/1b50671ca072dd646e98bee971a6d20d5c06a2de/code/modules/mob/living/carbon/human/species_types/plasmamen.dm#L20

Reply:
fixed thanks